.travis.yml

```diff
@@ -22,8 +22,10 @@ env:
     - CI_NODE_INDEX=3
     - CI_NODE_INDEX=4 KARMA="true" GITHUB_DEPLOY="true"
 
-before_install:
-  - ./script/upgrade_bundler.sh
+# Disabling due to travis errors.
+# See @oeoeaio d46888b
+#before_install:
+#  - ./script/upgrade_bundler.sh
```

Comment:
I'd rather remove the lines. That is what commits and their descriptions are for IMO.

Reply:
I'd rather merge rob's commit and remove this commit from this PR altogether.
I should add the 'wip' label :-)